app/code/Magento/Checkout/Test/Mftf/Test/CheckCheckoutSuccessPageTest.xml

```diff
@@ -27,10 +27,26 @@
         </before>
 
         <after>
+            <!--Cancel orders-->
+            <actionGroup ref="AdminLoginActionGroup" stepKey="adminLogin"/>
+            <actionGroup ref="AdminOrdersGridClearFiltersActionGroup" stepKey="goToOrdersPage"/>
+            <waitForLoadingMaskToDisappear stepKey="waitAfterClearFilters"/>
```

Comment:
If there's no such action - you can add it to `AdminOrdersGridClearFiltersActionGroup`

Reply:
That's done - I've added wait action in the end of `AdminOrdersGridClearFiltersActionGroup` and did small refactoring for tests where this action group is used.